pydantic_ai_slim/pydantic_ai/_utils.py

```
@@ -540,9 +540,25 @@ def get_traceparent(x: AgentRun | AgentRunResult | GraphRun[Any, Any, Any]) -> s
-    kv_pairs = (
-        f'{f.name}={getattr(self, f.name)!r}' for f in fields(self) if f.repr and getattr(self, f.name) != f.default
-    )
+    kv_pairs: list[str] = []
+    for f in fields(self):
+        if not f.repr:
+            continue
+        val = getattr(self, f.name)
+        if f.default is not MISSING:
+            try:
+                show = bool(val != f.default)
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)